src/Average of n number.md

```diff
@@ -0,0 +1,66 @@
+# Average of n numbers
+
+To calculate the average of n numbers
+
+## Result
+
+```js
+    Average of n numbers([1,2,2,3,3,4]); // '2.5'
```

Comment:
Let's keep this as a realistic function call

Reply:
I didn't get it what is meant by realistic function call?

Reply:
How your code actually will work, if someone wants to use it

In this case, e.g. Average([1,2,2,3,3,4]) // 2.5